src/internal/operators/delay.ts

```diff
@@ -41,8 +41,15 @@ import { timer } from '../observable/timer';
  * delayedClicks.subscribe(x => console.log(x));
  * ```
  *
- * @see {@link debounceTime}
  * @see {@link delayWhen}
+ * @see {@link throttle}
```

Comment:
there are a lot of formatation changes  in this pr that are entirely unrelated to this, I'd appreciate it if you could revert those

Reply:
The post-commit hook does that for me automatically, I did not want to interfere. Is there a way I can disable it? I ill revert those

Reply:
It got a bit messy, sorry for that. I will submit another PR shortly

Reply:
Here is the new PR, containing only the documentation change: https://github.com/ReactiveX/rxjs/pull/6156